run/markdown-preview/e2e_test.py

```diff
@@ -0,0 +1,91 @@
+# Copyright 2020 Google, LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import subprocess
+from urllib import request
+import uuid
+
+import pytest
+
+
+@pytest.fixture()
+def services():
+    # Unique suffix to create distinct service names
+    suffix = uuid.uuid4().hex
+
+    # Build and Deploy Cloud Run Services
+    subprocess.run(
```

Comment:
We probably need to make sure [`check=True`](https://docs.python.org/3/library/subprocess.html#subprocess.run) or that we check the error codes manually when these commands return.

Reply:
(same for all these calls below)